docs/contributors/releasing.md

```diff
@@ -27,6 +29,9 @@ title: Making a release
 
 ### Draft the release notes
 
+First of all, choose a new metal or an alloy as a new for the release! Use that
+in the release notes and in the release title on GitHub.
```

Comment:
New name for every minor (major.minor.patch) version? For each patch it might be too often 🤔

Reply:
@kpodsiad assuming changes we switch from the era `has new feature -> bump minor` to `almost all releases - bump patch`

Reply:
I was also thinking that we have plenty of metals and alloys so this might be more fun, but maybe if we bump minor more often then it's not needed. Will adjust the docs.

Reply:
adjusted

Reply:
hmm, maybe @tgodzik was right, new name for each release = more fun and because alloys are allowed, then there are plenty of names to choose from :D